test/e2e/cluster_upgrade.go

```diff
@@ -162,9 +160,12 @@ var _ = Describe("Skipped", func() {
 			v, err = realVersion(testContext.UpgradeTarget)
 			expectNoError(err)
 			Logf("Version for %q is %q", testContext.UpgradeTarget, v)
+		})
 
+		f := NewFramework("cluster-upgrade")
```

Comment:
Was going to ask if we really needed to split up the `BeforeEach`es like this, but this is code @mbforbes wrote. :\

@ihmccreery do you have any idea if we need to run `realVersion` before any of the Framework stuff starts?